spec/requests/providers_spec.rb

```diff
@@ -1451,6 +1451,23 @@ def gen_import_request
     end
   end
 
+  context 'cloud volume types subcollection' do
+    before do
+      @provider = FactoryBot.create(:ems_openstack)
```

Comment:
We try to not use specific provider types unless absolutely necessary.  Recommend:
```suggestion
      @provider = FactoryBot.create(:ems_storage)
```